apps/kg/dataloader/KGDataset.py

```diff
@@ -59,17 +59,22 @@ class KGDataset:
     The triples are stored as 'head_name\trelation_name\ttail_name'.
     '''
     def __init__(self, entity_path, relation_path,
-                 train_path, valid_path=None, test_path=None,
-                 format=[0,1,2], read_triple=True, only_train=False,
-                 skip_first_line=False):
-        self.entity2id, self.n_entities = self.read_entity(entity_path)
-        self.relation2id, self.n_relations = self.read_relation(relation_path)
-
-        if read_triple == True:
+                 train_path, valid_path=None, test_path=None, local2global_path=None,
```

Comment:
you need to explain the argument in the docstring.

Reply:
We will create a new class called PartitionKGDataset to handle the partition file.